package.json

```diff
@@ -174,6 +176,11 @@
             }
         ],
         "commands": [
+            {
+                "command": "vscode-kafka.open.docs.home",
+                "title": "Open VSCode Kafka Documentation",
```

Comment:
There's already a Kafka prefix:
<img width="736" alt="Screenshot 2021-01-25 at 14 29 59" src="https://user-images.githubusercontent.com/148698/105712249-d2c6d800-5f19-11eb-93bd-110d1c1529ec.png">

"Open Documentation" would be better